### PR TITLE
add flag argument to client.getPackages

### PR DESCRIPTION
**Pull request by ov3rk1ll**
As discussed in https://github.com/DeviceFarmer/adbkit/issues/55, this MR adds a parameter to the getPackages function that is passed to the `pm list packages` call.

### CODE REVIEW

File: lib/adb/client.ts [hunk 1]
```diff
@@ -233,10 +233,10 @@ class Client extends EventEmitter {
 			.nodeify(callback);
 	}
 
-	public getPackages(serial: string, callback?: Callback<string[]>): Bluebird<string[]> {
+	public getPackages(serial: string, flag?: string, callback?: Callback<string[]>): Bluebird<string[]> {
```
Comment (koral--) [inline lib/adb/client.ts:236]: Shouldn't it be called `flags`? It seems that multiple can be passed.

**Comment by koral--** [comment]
Please add a sign-off so DCO check can pass.

**Comment by drauggres** [comment]
FYI: this is breaking change. We should either change major version or keep compatibility (for example use other name for the method, e.g. `listPackages`).

File: lib/adb/client.ts (hunk 1 above)
Comment (ov3rk1ll) [reply lib/adb/client.ts:236]: You are right, you can provide multiple values so I'll change that to `flags`.

**Comment by ov3rk1ll** [comment]
> FYI: this is breaking change. We should either change major version or keep compatibility (for example use other name for the method, e.g. `listPackages`).

That is true, I'd suggest `getPackagesWithFlags` for a new function but `listPackages` also sounds okay to me

**Review by koral-- — APPROVED** [review]

**Pull request merged**